pkg/pgmodel/model/interface.go

```diff
@@ -59,6 +59,10 @@ func MetricTableName(conn pgxconn.PgxConn, metric string) (string, bool, error)
 		return "", true, err
 	}
 
+	if err := res.Err(); err != nil {
```

Comment:
If we are going to check `res.Err` twice (when res.Next() is false), why not check after line 50?

Reply:
The second check is if res.Next is true

Reply:
This is if Scan has an error